ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/radiobutton/v1/radiobutton/clientlibs/site/js/radiobuttonview.js

```diff
@@ -107,6 +107,16 @@
             });
         }
 
+        syncWidget() {
+            let widgetElementList =this.getWidget();
+            if(widgetElementList){
+                widgetElementList.forEach((widgetElement)=>{
+                    let idToReplace=widgetElement.id.split('_')[0];
```

Comment:
it is incorrect to assume any particular format for ids in markup. `underscore` in id here is an incorrect assumption

Reply:
Hi @nit23uec , 
This is not assumed for the generated ID, we have explicitly added this '_' in the markup.
https://github.com/adobe/aem-core-forms-components/blob/master/ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/radiobutton/v1/radiobutton/radiobutton.html#L44
cc:- @sakshi-arora1